src/components/CreateAccount.jsx

```diff
@@ -1,9 +1,285 @@
+                </Grid>
+
+                <Grid
+                  item
+                  lg={6}
+                  md={6}
+                  sm={12}
+                  xs={12}
+                  sx={{ display: "flex", justifyContent: "flex-end" }}
+                >
+                  <TextField
+                    label="Enter last name"
+                    variant="outlined"
+                    type="text"
+                    required
+                    className={classes.root}
+                      InputProps={{className: classes.input}}
+                    {...register("lastName", { required: true, maxLength: 15 })}
+                  ></TextField>
+                </Grid>
+
+                <Grid item lg={6} md={6} sm={12} xs={12}>
+                  <FormControl fullWidth>
+                    <InputLabel id="demo-simple-select-label">
+                      Select your gender
+                    </InputLabel>
+                    <Select
+                      labelId="demo-simple-select-label"
+                      id="demo-simple-select"
+                      label="Select your gender"
+                      className={classes.root}
+                      required
+                      inputProps={{className: classes.input}}
+                      {...register("gender")}
+                    >
+                      <MenuItem value={"male"}>Male</MenuItem>
+                      <MenuItem value={"female"}>Female</MenuItem>
+                    </Select>
+                  </FormControl>
+                </Grid>
+
+                <Grid
+                  item
+                  lg={6}
+                  md={6}
+                  sm={12}
+                  xs={12}
+                  sx={{ display: "flex", justifyContent: "flex-end" }}
+                >
+
+                  <FileBase64
+                    multiple={false}
+                    className={classes.imageSelectorStyle}
+                    {...register("profilePic")}
+                    onDone={({base64})=>{console.log(base64); setProfilePic(base64)}}
+
+                  />
+                </Grid>
+
+                <Grid item lg={12} md={12} sm={12} xs={12}>
+                  <TextField
+                    label="Enter your email"
+                    variant="outlined"
+                    type="email"
+                    className={classes.root}
+                    required
+                    InputProps={{className: classes.input}}
+                    {...register("email", { required: true })}
+                  ></TextField>
+                </Grid>
+
+                <Grid item lg={12} md={12} sm={12} xs={12}>
+                  <TextField
+                    label="Create password"
+                    variant="outlined"
+                    type="password"
+                    required
+                    className={classes.root}
+                    InputProps={{className: classes.input}}
+                    {...register("password", {
```

Comment:
please add capital letter and one special character validation

Reply:
Okay, sure !